### ISSUE TITLE
I can't see the colors of the themes well

### ISSUE THREAD

**Issue by MontejoJorge** [neovim/neovim#13142]
- `nvim --version`: v0.4.4
- `vim -u DEFAULTS` (version: ) behaves differently? no
- Operating system/version: Windows 10 x64
- Terminal name/version: cmd.exe (i try with alacritty and poweshell too)
- `$TERM`:

### Steps to reproduce using `nvim -u NORC`
```
nvim -u NORC
# Alternative for shell-related problems:
:colorscheme evening
```
### Actual behaviour
https://ibb.co/sgzbbSZ

### Expected behaviour
https://ibb.co/Sv4hFv0

I tried another Windows 10 system and I can see it properly. If it is a problem of some configuration I have not found the difference

**Comment by janlazo** [comment]
That is expected behavior and the colorschemes in the screenshots are different.

Truecolor colorschemes, like `gruvbox`, require `set termguicolors` before running the `:colorscheme` command and an interface that supports truecolor like a  GUI, ConEmu, or the new Windows 10 terminal.

**Comment by janlazo** [comment]
Default terminals for the command prompt (cmd.exe) and powershell.exe support 8-16 colors only.  You should not use truecolor colorschemes in those terminals. `:echo &t_Co` displays the number of colors supported in your environment. It must be display `256` for truecolor colorschemes to work.

**Comment by erw7** [comment]
> Default terminals for the command prompt (cmd.exe) and powershell.exe support 8-16 colors only.

Let me add a few things. The current version of Windows 10 `cmd.exe` and `powershell.exe` are available in 256 colors (and 24 bit color). I don't know which version of the software this change was made on, but it was a long time ago, so I don't think you're using that old a version. But, if 256 colors are not available in your environment with `cmd.exe`, your version of Windows 10 may be out of date.

> :echo `&t_Co` displays the number of colors supported in your environment. It must be display 256 for truecolor colorschemes to work.

It's a small detail, but the terminal emulator must support 24 bit color display to use `termguicolors`. Most modern terminal emulators support the display of 24 bit colors, but there are terminal emulators that only support the display of 256 colors. Thus, `termguicolors` may not be used even if `t_Co` returns 256.

**Comment by MontejoJorge** [comment]
@janlazo you are right, my windows 10 terminal shows only 8 colors. I have searched for the reason for this, I found that when they added the colors, they only did it for the new installations, so as not to overwrite the users' color settings. This color enhancement will only be available for new installations. One option if you don't want to or can't reinstall from 0 is to use the "ColorTool".

Thank you for your time

[Article of the color update](https://devblogs.microsoft.com/commandline/updating-the-windows-console-colors/#where-will-i-see-these-changes)

[Article of ColorTool](https://devblogs.microsoft.com/commandline/introducing-the-windows-console-colortool/)
